demo/src/Components/Pills/Pill.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { createElement as E } from 'react'
+import T from 'prop-types'
+import classNames from 'classnames'
+
+const Pill = (
+  {
+    node,
+    component,
+    children,
+    index,
+    activatePill,
+    isActive,
+    disabled,
+    className,
+    to,
+    type,
+    status,
+    ...rest
+  }) => {
+  const handleClick = () => { activatePill(index) }
+  const initial = 'c-pills__pill'
+  const optional = 'c-pills__pill--status'
+  const ComponentOverride = component
+  const enhancedClassName = classNames(
+    initial,
+    className,
+    { 'is-active': isActive },
+    { [`${optional} c-pills__pill--success`]: status === 'success' },
+    { [`${optional} c-pills__pill--info`]: status === 'info' },
+    { [`${optional} c-pills__pill--warning`]: status === 'warning' },
+    { [`${optional} c-pills__pill--error`]: status === 'error' },
+  )
+
+  if (ComponentOverride) {
+    return (
+      <ComponentOverride
+        to={to}
+        className={enhancedClassName}
+        {...rest}
+      >
+        {children}
+      </ComponentOverride>
+    )
+  }
+
+  return (
```

Comment:
Don't get too hooked up on allowing multiple node types for the demo here, it's just to literally test the styling, these React components here aren't being shipped - It's in Nebula-React where we should care about the React side of things.